velox/vector/VectorStream.h

```diff
@@ -49,6 +49,10 @@ class VectorSerializer {
 class VectorSerde {
  public:
   virtual ~VectorSerde() = default;
+  // Lets the caller pass a set of options to the Serde which are defined as a
+  // set of key-value pairs.
+  typedef std::unordered_map<std::string, std::string> Options;
```

Comment:
What's the advantage of using a map instead of a POD struct?

Reply:
moved this to using a POD struct as suggested